plenum/test/client/test_client_resends_not_confirmed_request.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+from plenum.common.constants import REPLY, OP_FIELD_NAME
+from plenum.common.types import f
+from plenum.test.spy_helpers import getAllArgs
+from stp_core.common.log import getlogger
+from stp_core.loop.eventually import eventually
+from plenum.test.client.conftest import passThroughReqAcked1
+from plenum.test.helper import stopNodes, send_signed_requests, \
+    waitForSufficientRepliesForRequests
+from plenum.test.malicious_behaviors_client import \
+    genDoesntSendRequestToSomeNodes
+
+logger = getlogger()
+
+nodeCount = 4
+clientFault = genDoesntSendRequestToSomeNodes("AlphaC")
+reqAcked1 = passThroughReqAcked1
+
+
+def fake_send(client):
```

Comment:
Do we need it just to make sure we don't have state proofs? Can we just set `nodes_wth_bls = 0`?